docs/docs/widgets/color-picker.md

```diff
@@ -3,19 +3,17 @@ id: color-picker
 title: Color Picker
 ---
 
-# Color Picker
-
-**Color Picker** widget is used to select the desired color from the color picker
+**Color Picker** component is used to select the desired color from the color picker
```

Comment:
Can you please add a full stop at the end of this sentence? Also please add the full stops wherever necessary.